cow_instrument/SpectrumInfo.h

```diff
@@ -30,59 +31,59 @@ template <typename InstTree> class SpectrumInfo {
    */
   SpectrumInfo(const DetectorInfo<InstTree> &detectorInfo)
       : m_detectorInfo(detectorInfo), m_l2(detectorInfo.size()),
-        m_spectra(std::move(detectorInfo.makeSpectra())) {
+        m_spectrum(std::move(detectorInfo.makeSpectra())) {
 
     initL2();
   }
 
   /**
    * @brief SpectrumInfo constructor. Suitable for 1:n spectra to detector
    * mapping.
-   * @param spectra : DetectorIndexes to SpectraIndexes
+   * @param spectrum : DetectorIndexes to spectrumIndexes
    * @param detectorInfo : DetectorInfo object
    */
-  SpectrumInfo(const std::vector<Spectrum> &spectra,
+  SpectrumInfo(const std::vector<Spectrum> &spectrum,
                const DetectorInfo<InstTree> &detectorInfo)
-      : m_spectra(spectra), m_detectorInfo(detectorInfo), m_l2(spectra.size()) {
+      : m_spectrum(spectrum), m_detectorInfo(detectorInfo), m_l2(spectrum.size()) {
     initL2();
   }
 
   void initL2() {
-    for (size_t spectraIndex = 0; spectraIndex < this->size(); ++spectraIndex) {
+    for (size_t spectrumIndex = 0; spectrumIndex < this->size(); ++spectrumIndex) {
 
       double l2Temp = 0;
-      for (auto detectorIndex : m_spectra[spectraIndex].detectorIndexes()) {
+      for (auto detectorIndex : m_spectrum[spectrumIndex].detectorIndexes()) {
         l2Temp += m_detectorInfo.l2(detectorIndex);
       }
       // Divide through by number of detectors
-      l2Temp /= m_spectra[spectraIndex].size();
-      m_l2[spectraIndex] = l2Temp;
+      l2Temp /= m_spectrum[spectrumIndex].size();
+      m_l2[spectrumIndex] = l2Temp;
     }
   }
 
-  size_t size() const { return m_spectra.size(); }
+  size_t size() const { return m_spectrum.size(); }
 
   size_t nDetectors() const {
     size_t count = 0;
-    for (const Spectrum &s : m_spectra) {
+    for (const Spectrum &s : m_spectrum) {
       count += s.size();
     }
     return count;
   }
 
   Spectrum spectra(size_t index) {
```

Comment:
This should be singular, i.e., (sorry for being picky, but I think it is important for readability)
```cpp
Spectrum spectrum(size_t index) {
   spectraRangeCheck(index, m_spectra);
   return m_spectra[index];
}
```